cocos/2d/renderer/mesh-buffer.ts

```diff
@@ -27,86 +27,79 @@
  * @packageDocumentation
  * @module ui
  */
-import { BufferUsageBit, MemoryUsageBit, InputAssemblerInfo, Attribute, Buffer, BufferInfo, InputAssembler } from '../../core/gfx';
+import { Device, BufferUsageBit, MemoryUsageBit, Attribute, Buffer, BufferInfo } from '../../core/gfx';
 import { ScalableContainer } from '../../core/memop/scalable-container';
-import { IBatcher } from './i-batcher';
+import { BufferAccessor } from './buffer-accessor';
+import type { LinearBufferAccessor } from './linear-buffer-accessor';
 import { getComponentPerVertex } from './vertex-format';
+import { warnID } from '../../core/platform/debug';
 
 export class MeshBuffer extends ScalableContainer {
     public static OPACITY_OFFSET = 8;
 
+    get accessor () { return this._accessor; }
```

Comment:
`_accessor` is a private instance of `BufferAccessor` class 
maybe this should be marked as `@internal`

interface with parameters of private objects should be marked as `@internal` too

Reply:
MeshBuffer won't have accessor later